File: src/targets/php/http2/fixtures/jsonObj-null-value.php
<?php

$client = new http\Client;
$request = new http\Client\Request;

$body = new http\Message\Body;
<<<<<<< HEAD
$body->append('{"foo":null}');

$request->setRequestUrl('https://httpbin.org/anything');
=======
$body->append(json_encode([
  'foo' => null
]));
$request->setRequestUrl('http://mockbin.com/har');
>>>>>>> upstream/master
$request->setRequestMethod('POST');
$request->setBody($body);

$request->setHeaders([
  'content-type' => 'application/json'
]);

$client->enqueue($request)->send();
$response = $client->getResponse();

echo $response->getBody();

File: src/targets/clojure/clj_http/fixtures/jsonObj-null-value.clj
(require '[clj-http.client :as client])

<<<<<<< HEAD
(client/post "https://httpbin.org/anything" {:content-type :json
                                             :form-params {:foo nil}})
=======
(client/post "http://mockbin.com/har" {:content-type :json
                                       :form-params {:foo nil}})
>>>>>>> upstream/master
